scene/main/viewport.cpp

```diff
@@ -884,6 +884,10 @@ void Viewport::_process_picking() {
 		}
 
 #ifndef _3D_DISABLED
+		if (is_input_handled()) {
```

Comment:
I believe, that this is too restrictive, since some users could depend on events getting to both, 2D nodes and 3D nodes. There is a configuration option to handle exactly this case: `physics_object_picking_first_only`.

```suggestion
		if (physics_object_picking_first_only && is_input_handled()) {
```